package/cfg/k3s-cis-1.23-permissive/node.yaml

```
@@ -103,53 +103,27 @@ groups:
 
       - id: 4.1.7
         text: "Ensure that the certificate authorities file permissions are set to 644 or more restrictive (Manual)"
-        audit: "stat -c %a /var/lib/rancher/k3s/server/tls/server-ca.crt"
+        audit: |
+          CAFILE=$(ps -ef | grep kubelet | grep -v apiserver | grep -- --client-ca-file= | awk -F '--client-ca-file=' '{print $2}' | awk '{print $1}' | uniq)
+          if test -z $CAFILE; then CAFILE=$kubeletcafile; fi
+          if test -e $CAFILE; then stat -c permissions=%a $CAFILE; fi
```

The change I proposed might fail for K3s, I recommend you keep the previous `audit` (based on [k3s-cis-1.7-permissive](https://github.com/rancher/security-scan/blob/e8dcabab764ea4915e34af1bdc206965ef25591a/package/cfg/k3s-cis-1.7-permissive/node.yaml#L99C9-L99C74)) but change ONLY the items (as you did here).

```suggestion
        audit: "stat -c %a /var/lib/rancher/k3s/server/tls/server-ca.crt"
```

---

done